bloom-housing/bloom
# feat: lottery toasts

This PR addresses #4223


## Description

Adds success and error toasts to the different steps of the lottery in partners.

## How Can This Be Tested/Reviewed?

Go through the lottery flow (run, re-run, release, retract, and publish) to see the new toasts in action. If you want to test an error toast, you can turn your network connection to offline in the developer tools network tab before clicking the button.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
